packages/dataclass-editor/src/master/ValidationRow.tsx

```diff
@@ -26,17 +16,26 @@ export const rowClassName = (messages: Array<MessageData>) => {
 type ValidationRowProps = {
   row: Row<DataClassField>;
   isReorderable: boolean;
+  updateOrder: (moveId: string, targetId: string) => void;
+  onClick?: React.MouseEventHandler<HTMLTableRowElement> | undefined;
+  onDrag?: React.DragEventHandler<HTMLTableRowElement> | undefined;
 };
 
-export const ValidationRow = ({ row, isReorderable }: ValidationRowProps) => {
-  const updateOrder = useUpdateOrder();
+export const ValidationRow = ({ row, isReorderable, updateOrder, onClick, onDrag }: ValidationRowProps) => {
   const messages = useValidation(row.original);
 
   const RowComponent = isReorderable ? ReorderRow : SelectRow;
 
   return (
     <>
-      <RowComponent id={row.index.toString()} row={row} className={rowClassName(messages)} updateOrder={updateOrder}>
+      <RowComponent
+        id={row.index.toString()}
+        row={row}
+        className={rowClassName(messages)}
+        updateOrder={updateOrder}
```

Comment:
I know that this does not come from you, but just realized it.
If you add this param to the SelectRow (e.g. if the sorting is enabled) then you get a warning...

Reply:
Yes you are right. Do I understand correctly that the only option now is to integrate the two individually, which means that there is significantly more code? Something like this:
```
{isReorderable ? (
<ReorderRow updateOrder={updateOrder}>
</ReorderRow>
) : (
<SelectRow>
</SelectRow>)
```

Reply:
yep. maybe we should ignore it for now